fluxoagil/__init__.py

```diff
@@ -0,0 +1,8 @@
+from flask import Flask
+
+def create_app():
+    app = Flask(__name__)
+    app.config['SECRET KEY'] = ''
+
+    return app
+    
```

Comment:
```suggestion
from flask import Flask
from flask_restful import Resource, Api

class HelloFluxoAgil(Resource):
    def get(self):
        return {'Welcome': 'to Fluxo Agil API'}

def create_app():
    app = Flask(__name__)
    api = Api(app)
    app.config['SECRET KEY'] = ''
    api.add_resource(HelloFluxoAgil, '/')
    return app
```

Aí fica com mais cara de API. Mas não vai ficar assim pra sempre